.github/ISSUE_TEMPLATE/docs_update.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
On second thought, let's remove the `docs_update.md` form and just stick with one bug report template. We can always beef up the form later if needed.

Reply:
Alright, I will be making the necessary changes.